test/e2e/storage/csi_volumes.go

```diff
@@ -58,6 +59,20 @@ var csiTestSuites = []func() testsuites.TestSuite{
 	testsuites.InitProvisioningTestSuite,
 }
 
+func csiTunePattern(patterns []testpatterns.TestPattern) []testpatterns.TestPattern {
+	tunedPatterns := []testpatterns.TestPattern{}
+
+	for _, pattern := range patterns {
+		// Skip inline volume tests
+		if pattern.VolType == testpatterns.InlineVolume {
```

Comment:
I just discussed with @davidz627, let's also skip the pre-provisioned case too because there's some 50-ish test cases.  sig-testing is encouraging moving provider-specific test cases out of tree, so if we decide to add test cases for pre-provisioned volumes in the future, we should do it out-of-tree.

Reply:
@msau42 

I agree with it. I will update the codes as so.